packages/terra-paginator/docs/ProgressiveREADME.md

```diff
@@ -0,0 +1,23 @@
+# Terra Paginator
+
+Paginator to be used for ordered data where the size of the data is known. Provides first, last, previous, next, and paged functionality.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-paginator`
+
+## Usage
+
+```jsx
+import React from 'react';
+import ProgressivePaginator from 'terra-paginator';
```

Comment:
We should update this dock to 

    import ProgressivePaginator from 'terra-paginator/lib/ProgressivePaginator';

Let's do that for the terra-site examples as well I think.

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/fd545ca9b18ee3883c109a58fe3be398ad95d18c